### DISCUSSION TITLE
Is there any size limitation for original images?

### DISCUSSION THREAD

**Discussion by ionutcirja** [vercel/next.js#71080] [Help]
### Summary

Hello,

In my website I have an image of which size is 12.4mb. Every time when the image is loaded vercel cache is MISS.
Is there any size limitation for images when using vercel?

Thank you,

### Additional information

It looks like an image of 9.9mb is cached.

**Comment by viii47** [comment]
Vercel's limit for images is 10MB in size and 8K (~67MP) in resolution — see https://vercel.com/docs/image-optimization/limits-and-pricing#limits
